src/test/regress/sql/create_role_propagation.sql

```diff
@@ -137,7 +137,7 @@ REVOKE dist_role_3 from dist_role_4 granted by test_admin_role;
 SELECT roleid::regrole::text AS role, member::regrole::text, (grantor::regrole::text IN ('postgres', 'non_dist_role_1', 'dist_role_1','test_admin_role')) AS grantor, admin_option FROM pg_auth_members WHERE roleid::regrole::text LIKE '%dist\_%' ORDER BY 1, 2;
 SELECT objid::regrole FROM pg_catalog.pg_dist_object WHERE classid='pg_authid'::regclass::oid AND objid::regrole::text LIKE '%dist\_%' ORDER BY 1;
 
-REVOKE dist_role_3 from non_dist_role_3 granted by test_admin_role;
```

Comment:
Rather than changing an existing test case that already works nicely before the fix, it'd be great to add a test that would fail on main but succeeds on this branch etc.

Reply:
This error is tricky actually. Since granted by statement is not being propagated, revoke statement never parsed before; therefore the error never exists. This appeared after I need to use granted by statement in revoke statement with cascade. Since there is no such test in main branch, there has been no errors actually.